build.gradle

```diff
@@ -90,7 +93,20 @@ githubPages {
 }
 
 shadowJar {
-    classifier = 'all'
+    baseName = 'launchdarkly-client'
+    //no classifier means that the fat + shaded jar becomes the default artifact
+    classifier = ''
+
+    // Shade all jars except for launchdarkly
+    relocate('com.google', 'com.launchdarkly.shaded.com.google')
```

Comment:
Unfortunately it doesn't look like we can automatically shade all but com.launchdarkly. If we add a net.something... dependency we'll need to add it here.